### PR TITLE
Closes #149

### LINKED ISSUE #149: proc_ssd updated clab data structure
I think proc_ssd should update the no of channels in dat.clab according to ssd output, since it has also a parameter which contains the original no of channels in dat.origClab. This update is necessary for the case when the data does not have full rank.

I think this should be fixed in --> line 97:   for k=1:Nc 
                                                or:    for k=1:size(dat.x,2)

### PR DESCRIPTION
**Pull request by IrinaNicolae**
If it's correct, I've modified proc_ssd to return updated clab structure as described in Issue #149 
 --> line 97: for k=1:size(dat.x,2)
Closes #149

### CODE REVIEW

**Comment by svendaehne** [comment]
Hi Irina,

It looks good to me. Except for one thing: could it be that now dat.clab contains new labels (for k <= n_components) as well as old labels (for n_components < k <= n_channels)?
I think it would make sense to newly initialize dat.clab before writing the new label names in it. 

Cheers,
Sven

**Comment by IrinaNicolae** [comment]
Ups, you are right. Thanks for checking. I hope it's in order, now.
Best,
Irina

File: processing/proc_ssd.m [hunk 1]
```diff
@@ -94,6 +94,7 @@
 
 %% rename channel labels and save old channel labels
 dat.origClab= dat.clab;
-for k=1:length(dat.clab)
+dat.clab={};
```
Comment (svendaehne) [inline processing/proc_ssd.m]: That looks better! Final suggestion: how about 
dat.clab = cell(1,size(dat.x,2)); 
?

**Comment by svendaehne** [comment]
Looks very good! Thank you and sorry for the delay.

**Pull request merged**